requirements.txt

```diff
@@ -66,4 +66,6 @@ werkzeug==0.16.1
 # It is not included by default because of the GPL license conflict.
 # ldap3==2.2.4
 Authlib==0.15.5
-advocate==1.0.0
\ No newline at end of file
+advocate==1.0.0
+nzpy>=1.15
+nzalchemy
```

Comment:
Query runner dependencies belong in `requirements_all_ds.txt`. `requirements.txt` is only for Redash's own dependencies.

You can see [this article](https://redash.io/help/open-source/dev-guide/write-a-query-runner#Finishing-up) for more info.

```suggestion
```